browser/src/VariantList/Variants.tsx

```diff
@@ -198,6 +206,38 @@ const Variants = ({
     table.current.scrollToDataRow(index)
   }, [positionLastClicked]) // eslint-disable-line react-hooks/exhaustive-deps
 
+  function getFirstIndexFromSearchText(
+    searchFilter: VariantFilterState,
+    variantSearched: Variant[],
+    variantsTableColumns: any
+  ) {
+    const searchedVariants = getFilteredVariants(searchFilter, variantSearched, variantsTableColumns)
+  
+    if (searchedVariants.length > 0) {
```

Comment:
I like this additional logic to get fix those edge cases, thanks for looking into that. When I locally try these changes, I'm now getting the search going far past the matched searched (I checked on Firefox and Chrome). Both if I'm before the search term, or after the search term, it scrolls far past the the highlighted variant (~4 full displayed sizes worth of variants). 

Any ideas as to what might be causing this? I haven't looked too far into it, and am happy to pair program and debug if that's helpful.